src/main/resources/db/changes/0010-add-available-column.yaml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-available-column
+      author: danetsao
+      changes:
+        - addColumn:
+            tableName: books
+            columns:
+            - column:
+                name: available
+                type: BOOLEAN
+                constraints:
+                  nullable: true
+                  defaultValueBoolean: true
```

Comment:
Ah good catch, would you mind editing the training document with that instead of `defaultValue`